<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementWidget</class>
 <widget class="QWidget" name="MeasurementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>777</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Measurement</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>2D Measurement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="measurement2DTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Vessel Wall Area / mm^2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Lumen Area / mm^2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>NMI</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Wall Thickness / mm</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Tract 1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tract 2</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>3D Measurement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="measurement3DTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <row>
         <property name="text">
          <string>Total Plaque /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Vessel Wall /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Lumen /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Calcification /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Hemorrhage /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LRNC /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LM /mm^3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Volume</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Patient Imformation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>MRI Imformation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="patientTableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>38</number>
          </attribute>
          <row>
           <property name="text">
            <string>Name</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ID</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>DOB</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Gender</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>(who)</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>???????</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>yyyymmdd</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>(M/F/Trans)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="MRITableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>38</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <row>
           <property name="text">
            <string>ScanDate</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ReportDate</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>L/RSide</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Dr'sName</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>yyyymmdd</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>yyyymmdd</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>(L/R)</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>Dr. (who)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>161</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>STENOSIS %</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="stenosisSpinBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>label_2</zorder>
     <zorder>measurement2DTableWidget</zorder>
     <zorder>label_3</zorder>
     <zorder>measurement3DTableWidget</zorder>
     <zorder>verticalSpacer</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
